ee/settings.py

```diff
@@ -26,5 +26,9 @@
     ].split(",")
 
 # ClickHouse and Kafka
-CLICKHOUSE_DENORMALIZED_PROPERTIES = os.getenv("CLICKHOUSE_DENORMALIZED_PROPERTIES", "").split(",")
+CLICKHOUSE_DENORMALIZED_PROPERTIES = (
+    os.getenv("CLICKHOUSE_DENORMALIZED_PROPERTIES").split(",")
+    if os.getenv("CLICKHOUSE_DENORMALIZED_PROPERTIES", False)
+    else False
```

Comment:
I think we always want a list (empty, or with valid strings), so that the `in` operator in `json_extract` works